imap_processing/tests/test_cli.py

```diff
@@ -107,56 +107,37 @@ def test_codice(mock_codice_l1a, mock_instrument_dependencies):
     assert mocks["mock_upload"].call_count == 1
 
 
-@mock.patch("imap_processing.cli.hi_l1a.hi_l1a")
-def test_hi_l1a(mock_hi_l1a, mock_instrument_dependencies):
+@pytest.mark.parametrize("data_level, n_prods", [("l1a", 2), ("l1b", 1), ("l1c", 1)])
```

Comment:
Nice use of parametrization here! I've been wondering if we could parameterize over some things in this file with all the duplication.